test/test-unit/broken_unicode.test

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Test is needed to check how test-run handle
+# tests with result files. Since commit
+# 395edeb6b743c4479a62dd2183062124973d2b2a
+# 'python3: decouple bytes and strings' test-run
+# reads test output and result files in byte mode.
+# Output of this test contains broken unicode sequences
+# and test-run will fail if it will try to decode output.
+
+printf 'TAP version 13\n'
+printf '1..2\n'
```

Comment:
Should be `1..3`?

Reply:
Fixed

Reply:
It seems, the result file was not updated.